TARS/src/mcu_main/hilsim/hilsimpacket.proto

```diff
@@ -0,0 +1,45 @@
+/**
+ * @struct HILSIMPacket
+ * @brief Structure to hold data received serially from a desktop computer
+ *
+ * The simulated/past-launch data is streamed through serial row-by-row to TARS. TARS receives it in the HILSIM thread and populates
+ * data that would otherwise be read from sensors via the received HILSIM packet. Used for rapid testing and iteration of onboard
+ * hardware, GNC, and telemetry systems.
+ * To generate hilsimpacket.pb.h, use the following command:
+ * `python nanopb_generator/nanopb_generator.py hilsimpacket.proto`
+ * To generate hilsimpacket_pb2.py, use the following command:
+ * `protoc -I=. --python_out=. hilsimpacket.proto`
+*/
+syntax = "proto2";
+
+// TODO(Zyun): Might be more optimal to change all of this to required because protobuf for platformio
+// adds an extra bool between each data package, which could lead to some data wastage
+message HILSIMPacket {
+    // High-G IMU data
+    optional float imu_high_ax = 1;
```

Comment:
Can we get a comment to specify why the `optional` keyword is used here. It isn't immediately obvious to people that haven't worked extensively with pb (aka myself hehe)